# Update rate in Hz
update_rate: 100

# Switch expected update rate in Hz
switch_expected_update_rate: 30

# Switch update lag tolerance in Hz
switch_update_lag_tolerance: 10

# Switch startup timeout in seconds
switch_startup_timeout: 20

# Landing detected height threshold
landing_detected_height: 0.04

# Takeoff detected height threshold
takeoff_detected_height: 0.09

# Override switches on takeoff height theshold
takeoff_detected_override_height: 0.45

# Whether or not to use the switches
use_switches: false

# Minimum velocity threshold for landing
min_velocity_threshold: 0.1

# Minimum velocity threshold duration
min_velocity_duration: 2

# Height at which velocity detector is active
velocity_detector_height: 0.1
